### ISSUE TITLE
Screen reader accessibility of the web app in particular needs refining

### ISSUE THREAD

**Issue by Almost-Senseless-Coder** [linagora/tmail-flutter#4334]
### Description

The overall accessibility for screen reader users of the web app needs refining. Emails in inboxes are just marked by single letters, lots of unlabelled images unlabelled icons, etc.

The situation is a bit better on iOS, but there, too, are serious issues, such as in the formatting menu of the compose view.

### Expected result

Screen readers should at least be able to return meaninful information about all relevant UI elements. Irrelevant UI elements, such as avatars, should be hidden or at least receive a meaninful alt text.

### Current behavior

Lots of unlabelled stuff. Makes navigation quite hard.

#### Reproduction Steps

Open the Twake mail app on iOS/Android or the tmail-web frontend and turn a screen reader of your choice on, such as VoiceOver, TalkBack, NVDA; you'll notice the issues I mentioned. In fact, I thinn automated accessibility testing tool would spot many of them.k 

### Acceptance criteria

The list of acceptance criteria (defined in the corresponding User Story or coming from your functional knowledge of the product)
These acceptance criteria must be met by the corrective implementation done by the developers.

Ideally, the app should meet WCAG AA level at some point; AAA would be amazing.

### Context

Plattform and device-independent, though iOS is in a better state than the web frontend.

**Comment by hoangdat** [comment]
hi @poupotte can we add it into our roadmap?
